### PR TITLE
NEWS.md: add release notes for v0.53.0

### PR DESCRIPTION
**Pull request by grondo**
Feel free to amend as necessary. I've already added #5359 and #5360 assuming they'll be merged before EOD.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+ * resource: fix problem with exclusions when R is dynamically discovered
+   (#5339)
+ * python: clear KVS txns on commit error (#5335)
+ * python: do not return int status in kvs functions (#5329)
+ * python: fix exists in KVSDir with initial paths (#5331)
+ * python: fix writes in KVSDir with initial paths (#5322)
+ * flux-job: fix invalid --original info output (#5318)
+ * flux-job: fix `flux job status` handling of nonfatal exceptions (#5320)
+ * job-manager: fix prolog/epilog exception handling (#5321)
+ *  job-info: ignore duplicate lookup keys (#5317)
```
Comment (chu11) [inline NEWS.md:36]: errant space?

Comment (grondo) [reply NEWS.md]: Yeah, I guess the `src/test/relnotes.sh` script preserves leading spaces in PR/merge commit titles.

**Review by garlick — APPROVED** [review]
LGTM!

**Comment by grondo** [comment]
Thanks, setting MWP.

**Pull request merged**